### DISCUSSION TITLE
Duplicate product

### DISCUSSION THREAD

**Discussion by jclee100** [lunarphp/lunar#1106] [Feature Requests]
Please add a feature to allow admin to duplicate a product.

For example when creating a new product such as a t-shirt with a different print, the admin has to key in the brand, tax, price, weight, 
 and dimensions when it's exactly the same as a existing product. A feature to duplicate product saves a lot of effort.

**Comment by alecritson** [comment]
Hey @jclee100 Yes this something we want to add at some point as it's quite useful.
